weasyl/define.py

```diff
@@ -1081,6 +1088,9 @@ def cdnify_url(url):
 
 
 def get_resource_path(resource):
+    if reload_templates:
```

Comment:
Maybe, but I couldn’t come up with a better name. What describes the templates and the resources but not the non-template Python?